### DISCUSSION TITLE
Allow for `virtual` properties

### DISCUSSION THREAD

**Discussion by rsking** [canton7/PropertyChanged.SourceGenerator#27] [Ideas] [answered]
This is important for some database access tools, such as `NHibernate`.

These tools require properties to be virtual to operate.

**Comment by canton7** [comment, marked as answer]
It's normally a good idea to separate your database models and ViewModels -- re-using them may sound like a good idea to start with, but it quickly gets annoying, and you end up needing to separate them anyway.

That said, this shouldn't be hard.

**Reply by canton7** [reply]
https://github.com/canton7/PropertyChanged.SourceGenerator/commit/63feeecc2a9a2d4a8a682094f631c28c87c38996

**Reply by rsking** [reply]
I actually do have these as a model, but wanted to have the ability to have a model display changes in the UI. Again, kinda working as a view model, but these are purely domain models representing tables in the database.

Thanks for the update.
